docs/restore-individual-tables.md

```diff
@@ -1,76 +1,101 @@
 ```
 
+## Save the schema
+
+Before importing a table, you must have the schema for all tables you intend to restore. XtraBackup does not back up the table definitions, only the data files. If the original schema is lost, MySQL will not recognize the restored .ibd files.
+
+When you take a backup, you might not know which tables are included. It’s important to save the schema for all tables so you can restore them later without issues.The schema can change between the time you take the backup and when you run mysqldump, so make sure to capture the latest structure. 
+
+You could also use the script in the [Dump schema for all tables](dump-schema.md). You must modify the script for your specific use case and environment.
+
+This ensures you have the necessary table structures when you restore the backup.
+
 ## Export the table
 
-To generate an `.ibd` file in the target directory, create the table using the `innodb_file_per_table` mode:
+First, you need to create the table using what's called `innodb_file_per_table` mode. This creates an `.ibd` file in your source directory. You can check if the file exists by running:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-During the `--prepare` step, add the [--export] option to the
-command. For example:
+Next, when you run the `--prepare` step, add the [--export] option to your command. This tells XtraBackup to create the special files needed for moving tables:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-When restoring an encrypted InnoDB tablespace table, add the keyring file:
+If your table is encrypted (has extra protection), you'll need to include the keyring file too:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/tmp/table \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
 ```
 
-The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the `.cfp` file, which contains the transfer key and an encrypted tablespace key.
-
-The files are located in the target directory:
+After running these commands, you'll have two important files in your source directory. These are the only files you need to import the table:
 
 ```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
+/data/backups/mysql/test/export_test.cfp
 ```
 
 ## Import the table
 
-Import the table by doing the following:
+Now let's move the table to its new home. To import the table, on the target server,you need to do the following:
 
-* Create a copy of the table
-* Discard the table's original tablespace
-* Copy the `.ibd` and `.cfg` files to the location
-* Import the tablespace
+* Use the schema and create an empty copy of the table in the new location.
```

Comment:
The entire point of the script is to use it for restore. Let's elaborate on it a bit more. Can you please refine this text and add it?

"

or simply say "If you used the script (dump-schema.md) during the backup, you have saved the table definitions. You may use the appropriate table .sql file to create the table."

I guess this one line adds more value:
mysql -uroot < ./schema_dumps/test2/create_database_test2.sql

Reply:
What text are you asking about?

Reply:
@patrickbirch Am referring to "* Use the schema and create an empty copy of the table in the new location."

We should elaborate it a bit. We have saved the dumps earlier.